app/navigation/root-navigator.tsx

```diff
@@ -101,6 +101,12 @@ import {
 import { usePersistentStateContext } from "@app/store/persistent-state"
 import { NotificationSettingsScreen } from "@app/screens/settings-screen/notifications-screen"
 import { WelcomeFirstScreen } from "../screens/welcome-screen"
+import {
+  ReconciliationReport,
+  AggregationReport,
+  TopRecipientReport,
+  TopSenderReport,
```

Comment:
Remove AggregationReport, TopRecipientReport and TopSenderReport since there is no file for them and not used